azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/models/CallCompositeTelecomManagerOptions.java

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.calling.models;
+
+/**
+ * CallCompositeTelecomManagerOptions {@link CallCompositeTelecomManagerOptions}.
+ */
+public final class CallCompositeTelecomManagerOptions {
+    private final CallCompositeTelecomManagerIntegration telecomIntegration;
+    private String phoneAccountId;
+
+    /**
+     * Creates a CallCompositeTelecomManagerOptions.
+     *
+     * @param telecomIntegration The telecom integration.
+     * @param phoneAccountId A string identifier that is unique across PhoneAccountHandles with the
+     *                       same component name. Apps registering PhoneAccountHandles should ensure
+     *                       that the ID provided does not expose personally identifying information.
+     *                       A ConnectionService should use an opaque token as the PhoneAccountHandle identifier.
+     *                       Note: Each String field is limited to 256 characters.
+     */
+    public CallCompositeTelecomManagerOptions(final CallCompositeTelecomManagerIntegration telecomIntegration,
+                                              final String phoneAccountId) {
```

Comment:
`phoneAccountId` should not be a required field. If Contoso using `APPLICATION_IMPLEMENTED_TELECOM_MANAGER` then they do not need to provide `phoneAccountId`